src/Console/Command/WatchCommand.php

```diff
@@ -92,6 +115,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $criteria = $criteriaBuilder->build();
 
+        $errOutput = $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output;
```

Comment:
Why are we passing only the error output here?

Reply:
Like with Symfony, I'd prefer to have logs wrote on STDERR rather than STDOUT (fallback to)

Reply:
I also checked, this is also done in Symfony Messenger. Everything is written to std error there. I honestly don't know when I would write something to std error or std out, so I had everything in std out. But I think it makes sense to put logs in std error, at least for logs. 👍